[MM-205]: removed default values from Input element in create_issue.jsx

Kshitij-Katiyar opened the pull request:
#### Summary
Removed default values from create_issue.jsx

#### Ticket Link
  Fixes https://github.com/mattermost/mattermost-plugin-github/issues/205

wiggin77 commented:
@Kshitij-Katiyar I'm not clear how removing a `disabled=false` has anything to do with default values.  Can you explain?

Kshitij-Katiyar commented:
> @Kshitij-Katiyar I'm not clear how removing a `disabled=false` has anything to do with default values. Can you explain?

I looked into this [convo](https://github.com/mattermost/mattermost-plugin-github/pull/123/files/cc4ed0749ae13f9c7ea2a71592e666d7cbfd22aa#r390700807) and removed the values which were already defined as default prop values inside the Input element.
The PR title was unclear but I have updated it to more clarity
